portfolio/src/main/webapp/index.html

```diff
@@ -2,70 +2,95 @@
 <meta charset="utf-8">
 <title>My Portfolio</title>
 <link rel="stylesheet" href="style.css">
-<div id="header-content">
-  <div id="image-container">
-    <img src="images/main-selfie.jpg" alt="Picture of me" id="header-selfie">
-    <button id="cycle-button" onclick="cyclePictures()"> Cycle Pictures </button>
+<div id="greeting-container">
+  <h1>Malachi's portfolio</h1>
+  <h3>My name is Malachi Brewer.
+      I am a sophomore at the University of Georgia.</h3>
+  <h3>Welcome Buds! I created this website to house 
+      professional and academic information about myself.
+      Clicking on the buttons below will take you to 
+      different parts of my life. </h3>
+</div>
+<div id="layout-container">
+  <div id="nav-container">
+      <ul id="nav-bar">
+        <li data-target-id="about-me">About Me
+        <li data-target-id="education">Education
+        <li data-target-id="professional">Professional
+        <li data-target-id="facts">Random Facts
+        <li data-target-id="gallery">Photo Gallery
+      </ul>
+  </div>
+  <div id="content-container">
+    <article id="about-me" class="content-article hidden">  
+	  <h3 class="content-title" id="about-me-title">About Me
+		<!--instragram.svg was donloaded form flaticon.com and authored by Dave Gandy-->
```

Comment:
nit: "downloaded"

Reply:
Thanks. I thought I fixed this in the last commit

Reply:
Thanks!